### PR TITLE
Chore: useFlowRunGraphMock, useTaskRunMock

### PR DESCRIPTION
**Pull request by znicholasbrown**
^Also adds a flow run radar demo

### CODE REVIEW

**Comment by pleek91** [comment]
Seeing something in this PR that we haven't done before. Not necessarily against it but wanna make sure I understand what's happening.  You added a `FlowRunGraphMock` type and use that for the seed type. And then in the mock api you use the `get` method to grab an entire graph. 

Assuming I understand that correctly. Is that needed for anything other than convenience/performance of getting a graph? Or is that also because there's not a way to associate graph nodes together?

File: src/mocks/flowRunGraph.ts [hunk 1]
```diff
@@ -15,29 +16,30 @@ export type UpstreamReference = {
...
 
   // Create nodes
+  let i = 0
   while (nodes.length < size) {
-    const target: GraphNode = this.create('graphNode')
+    const target: GraphNode = this.create('graphNode', [{ id: ids[i] }])
 
     const proxy = new Proxy(target, {})
     nodes.push(proxy)
+    ++i
```
Comment (pleek91) [inline src/mocks/flowRunGraph.ts:42]: What is the proxy for?

**Comment by znicholasbrown** [comment]
> Assuming I understand that correctly. Is that needed for anything other than convenience/performance of getting a graph? Or is that also because there's not a way to associate graph nodes together?

Yup it's exactly the latter; in the API the `/graphs` route returns no associating information at the top level so we'd need to implicitly rely on the shape of its items in a serialization mock like this.

**Review by pleek91 — APPROVED** [review]
LGTM

**Pull request merged**